packages/connect/src/index.ts

```diff
@@ -3,31 +3,19 @@
  *
  * Connecting to a chain is done in two steps:
  *
- * 1. Call {createScClient} or {createPolkadotJsScClient}, which gives you a so-called *client*.
+ * 1. Call {createScClient}, which gives you a so-called *client*.
  * 2. Call `addChain` or `addWellKnownChain` on this client.
  *
- * Use {createPolkadotJsScClient} if you want to bind the PolkadotJS library to substrate-connect.
- * This client will provide chains that implement the `ProviderInterface` interface defined in the
- * `@polkadot/api` NPM package. These chains can be passed to `ApiPromise.create`, and you can
- * then use the PolkadotJS library as you normally would. See <https://polkadot.js.org/docs>.
- *
  * Use {createScClient} if you are interested in a lower-level experience where you directly
  * send JSON-RPC requests and receive responses.
- * In the library's internals, {createPolkadotJsScClient} is implemented on top of
- * {createScClient}.
- *
- * If you use {createPolkadotJsScClient}, be aware that the PolkadotJS library and its API are
- * fundamentally built around full nodes functionnalities, while substrate-connect is a light
- * client. Some functionnalities of PolkadotJS will not work properly, and/or not as well as they
- * could. This is fundamentally not fixable, and, while {createPolkadotJsScClient} is acceptable
- * for demos and prototypes, it is not possible to achieve the best user experience by using
- * PolkadotJS. Proper light-client-oriented high-level libraries built on top of
+ *
+ * Proper light-client-oriented high-level libraries built on top of
  * {createScClient} are currently in development.
```

Comment:
That's now weirdly out of context. The word "proper" is here because we talk about the problems of PolkadotJS right above.

I would still mention to users that they can use PolkadotJS, otherwise they're lost in nature

Reply:
Yep, I agree.